transformer_engine/jax/setup.py

```diff
@@ -29,11 +29,11 @@
 
 
 from build_tools.build_ext import get_build_ext
-from build_tools.utils import copy_common_headers, install_and_import
+from build_tools.utils import copy_common_headers, install_packages
 from build_tools.te_version import te_version
 from build_tools.jax import setup_jax_extension
 
-install_and_import("pybind11")
+install_packages("pybind11")
```

Comment:
We should be able to remove this since Pybind11 is included in the `pyproject.toml`:
```suggestion
```

Reply:
This is a standalone setup files for wheels that isn't using in the normal build process. pyproject.toml is only for the top level file.